### PR TITLE
Redo docs on operation translation

### LINKED ISSUE #56: Comprehensive function translation docs
As in https://github.com/dotnet/EntityFramework.Docs/pull/2744

### PR DESCRIPTION
**Pull request by roji**
Closes #56

/cc @bricelam @ajcvickers @smitpatel who are involved in this on the EF Core side
/cc @Brar who I know likes docs :)

### CODE REVIEW

File: conceptual/EFCore.PG/mapping/full-text-search.md [hunk 1]
```diff
@@ -159,3 +143,19 @@ var npgsql = context.Products
         .Matches("Npgsql"))
     .ToList();
 ```
+
+## Operation translation
+
+Almost all PostgreSQL full text search functions can be called through LINQ queries. All supported EF Core LINQ methods are defined in extension classes in the `Microsoft.EntityFrameworkCore` namespace, so simply referencing the Npgsql provider will light up these methods. Here is a table showing translations for some operations; if an operation you need is missing, please open an issue to request for it.
+
+C#                                                                          | .NET
```
Comment (YohDeadfall) [inline conceptual/EFCore.PG/mapping/full-text-search.md:151]: Not sure that .NET supports that expressions. The same below.

File: conceptual/EFCore.PG/mapping/json.md [hunk 2]
```diff
@@ -198,31 +198,31 @@ The following expression types and functions are translated:
...
-| `.Where(e => EF.Functions.JsonExists(e.Customer, "Age")`                                  | [`WHERE "Customer" ? 'Age'`](https://www.postgresql.org/docs/current/functions-json.html#FUNCTIONS-JSONB-OP-TABLE)
-| `.Where(e => EF.Functions.JsonExistsAny(e.Customer, "Age", "Address")`                    | [`WHERE "Customer" ?\| ARRAY['Age','Address']`](https://www.postgresql.org/docs/current/functions-json.html#FUNCTIONS-JSONB-OP-TABLE)
-| `.Where(e => EF.Functions.JsonExistsAll(e.Customer, "Age", "Address")`                    | [`WHERE "Customer" ?& ARRAY['Age','Address']`](https://www.postgresql.org/docs/current/functions-json.html#FUNCTIONS-JSONB-OP-TABLE)
-| `.Where(e => EF.Functions.JsonTypeof(e.Customer.GetProperty("Age") == "number")`          | [`WHERE jsonb_typeof("Customer"->'Age') = 'number'`](https://www.postgresql.org/docs/current/functions-json.html#FUNCTIONS-JSON-PROCESSING-TABLE)
+C#                                                                          | .NET
```
Comment (YohDeadfall) [inline conceptual/EFCore.PG/mapping/json.md:215]: Here.

File: conceptual/EFCore.PG/mapping/nts.md [hunk 3]
```diff
@@ -122,3 +78,47 @@ public class City
...
 
 Once you do this, your column will be created as `geography`, and spatial operations will behave as expected.
+
+## Operation translation
+
+The following table lists NetTopologySuite operations which are translated to PostGIS SQL operations. This allows you to use these NetTopologySuite methods and members efficiently - evaluation will happen on the server side. Since evaluation happens at the server, table data doesn't need to be transferred to the client (saving bandwidth), and in some cases indexes can be used to speed things up.
+
+Note that the plugin is far from covering all spatial operations. If an operation you need is missing, please open an issue to request for it.
+
+C#                               | .NET
```
Comment (YohDeadfall) [inline conceptual/EFCore.PG/mapping/nts.md:88]: Here too.

File: conceptual/EFCore.PG/mapping/range.md [hunk 4]
```diff
@@ -56,19 +56,17 @@ This will translate to an SQL operation using the PostgreSQL `@>` operator, eval
...
-| `.Where(c => c.SomeRange.DoesNotExtendRightOf(otherRange))`| [`WHERE x."SomeRange" <& @__otherRange_0`](https://www.postgresql.org/docs/current/static/functions-range.html#RANGE-OPERATORS-TABLE)
-| `.Where(c => c.SomeRange.IsAdjacentTo(otherRange))`        | [`WHERE x."SomeRange" -\|- @__otherRange_0`](https://www.postgresql.org/docs/current/static/functions-range.html#RANGE-OPERATORS-TABLE)
-| `.Select(c => c.SomeRange.Union(otherRange))`              | [`SELECT x."SomeRange" + @__otherRange_0`](https://www.postgresql.org/docs/current/static/functions-range.html#RANGE-OPERATORS-TABLE)
-| `.Select(c => c.SomeRange.Intersect(otherRange))`          | [`SELECT x."SomeRange" * @__otherRange_0`](https://www.postgresql.org/docs/current/static/functions-range.html#RANGE-OPERATORS-TABLE)
-| `.Select(c => c.SomeRange.Except(otherRange))`             | [`SELECT x."SomeRange" - @__otherRange_0`](https://www.postgresql.org/docs/current/static/functions-range.html#RANGE-OPERATORS-TABLE)
+C#                                                                          | .NET
```
Comment (YohDeadfall) [inline conceptual/EFCore.PG/mapping/range.md:59]: Once more.

**Review by YohDeadfall — APPROVED** [review]

**Comment by roji** [comment]
Thanks @YohDeadfall, will leave open to see if the EF folk have anything to say.

**Review by bricelam — APPROVED** [review]
LGTM

Note, I put the *Added in* column in the EF docs mostly because we have older, LTS versions. Not sure if it's as relevant here.

**Pull request merged**